tests/functional/oauth/test_oauth.py

```diff
@@ -88,3 +90,28 @@ def models(self):
     def test_snowflake_basic(self, project):
         run_dbt()
         check_relations_equal(project.adapter, ["MODEL_3", "MODEL_4"])
+
+
+class TestSnowflakeOAuthExpiration:
```

Comment:
Looking at the test run logs I think the issue here is that as part of test setup we are creating a connection with the database so it's failing in test setup and not in the `test_token_expiration` method

Reply:
updated the test, it hits the error then for some reason continues on into a RunTimeError as well